src/ScottPlot/Plottable/RadialGaugePlot.cs

```diff
@@ -0,0 +1,709 @@
+        /// Default value is set to <see langword="False"/>.
+        /// </summary>
+        public bool NormBackGauge { get; set; } = false;
+
+        /// <summary>
+        /// Angle (in degrees) at which the gauges start: 270° for North (default value), 0° for East, 90° for South, 180° for West, and so on.
+        /// Expected values in the range [0°-360°], otherwise unexpected side-effects might happen.
+        /// </summary>
+        public float StartingAngleGauges
+        {
+            get => _StartingAngleGauges;
+            set
+            {
+                _StartingAngleGauges = (float)ReduceAngle(value);
+                ComputeAngularData();
+            }
+        }
+        private float _StartingAngleGauges = 270f;
+
+        /// <summary>
+        /// The initial angle (in degrees) where the background gauges begin. Default value is 270° the same as <see cref="StartingAngleGauges"/>.
+        /// </summary>
+        public float StartingAngleBackGauges
+        {
+            get => _StartingAngleBackGauges;
+        }
+        private float _StartingAngleBackGauges = 270f;
+
+        /// <summary>
+        /// The empty space between gauges as a percentage of the gauge width.
+        /// Values in the range [0-100], default value is 50 [percent]. Other values might produce unexpected side-effects.
+        /// </summary>
+        public float GaugeSpacePercentage
+        {
+            get => _GaugeSpacePercentage;
+            set => _GaugeSpacePercentage = (value > 100 ? 100 : (value < 0 ? 0 : value));
+        }
+        private float _GaugeSpacePercentage = 50f;
+
+        /// <summary>
+        /// <see langword="Color"/> of the value labels drawn inside the gauges.
+        /// </summary>
+        public Color GaugeLabelsColor { get; set; } = Color.White;
+
+        /// <summary>
+        /// Size of the gague label text as a percentage of the gauge width.
+        /// Values in the range [0-100], default value is 75 [percent]. Other values might produce unexpected side-effects.
+        /// </summary>
+        public float GaugeLabelsFontPct
+        {
+            get => _GaugeLabelsFontPct;
+            set => _GaugeLabelsFontPct = (value > 100 ? 100 : (value < 0 ? 0 : value));
+        }
+        private float _GaugeLabelsFontPct = 75f;
+
+        /// <summary>
+        /// <see langword="Font"/> used for labeling values on the plot
+        /// </summary>
+        public Drawing.Font Font { get; set; } = new();
+
+        /// <summary>
+        /// <see langword="True"/> if value labels are shown inside the gauges.
+        /// Size of the text is set by <see cref="GaugeLabelsFontPct"/> and color by <see cref="GaugeLabelsColor"/>.
+        /// </summary>
+        public bool ShowGaugeValues { get; set; } = true;
+
+        public System.Drawing.Drawing2D.LineCap EndCap { get; set; } = System.Drawing.Drawing2D.LineCap.Triangle;
```

Comment:
This is a nice way of doing this.

Reply:
@arthurits this is awesome! I wouldn't have thought of this 🚀